subt_ign/launch/competition.ign

```diff
@@ -138,6 +140,7 @@
 
   MARSUPIAL_PARENT_ROBOT_POSITION_OFFSETS = {
     "X1" => Vector3d.new(0, 0, 0.717),
+    "MARBLE_HUSKY" => Vector3d.new(-0.06, 0, 0.392),
```

Comment:
looks like the offset needs to be higher. The UAVs are currently overlapping with the landing pad. Most obvious when testing with X3:

![marble_husky_x3_marsupial](https://user-images.githubusercontent.com/4000684/124507575-e60c8800-dd82-11eb-8e9a-b6e7b9c6095f.png)

Reply:
@bfotheri pinging on the above feedback

Reply:
The offsets were based on the QAV500.  Also, it looks like your visualization of the Husky didn't use the updated sdf?  The black platform should look different.

Reply:
I believe the black platform is updated for sensor config 5 but that is how it looked originally for the computing block.

Reply:
I think the model on Fuel may not have updated. I can fix that. I'm now testing with `localModel:=true`. This is what I see:

![husky_c5_marsupial](https://user-images.githubusercontent.com/4000684/124841492-3847d780-df42-11eb-9db0-c1a15052c073.png)

Is this the updated mesh? There is still a light overlap between the two.

Reply:
That does appear to be the updated mesh. I'll push a commit to fix that offset and the static_update_frequency param

Reply:
Commit pushed.